cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,247 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { PLUGIN_NAME } from "../support/constants";
+import sampleTransform from "../fixtures/sample_transform";
+
+const TRANSFORM_ID = "test_transform_id";
+
+describe("Transforms", () => {
+    beforeEach(() => {
+      cy.deleteAllIndices();
+
+      // Set welcome screen tracking to test_transform_target
+      localStorage.setItem("home:welcome:show", true);
+
+      // Go to sample data page
+      cy.visit(`${Cypress.env("opensearch_dashboards")}/app/home#/tutorial_directory/sampleData`);
+
+      // Click on "Sample data" tab
+      cy.contains("Sample data", { timeout: 20000 }).click({ force: true });
+      // Load sample eCommerce data
+      cy.get(`button[data-test-subj="addSampleDataSetecommerce"]`).click({ force: true });
+
+      // Verify that sample data is add by checking toast notification
+      cy.contains("Sample eCommerce orders installed", { timeout: 60000 });
```

Comment:
I don't think we want to actually test does going to the sample data tab works etc, we just want the data in the cluster before we start out test, so could we just make the API call directly from here and skip the browser interaction?

(i.e. whatever API is being called for loading the sample data, just call it directly)